net/busybox-httpd/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2007-2021 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=busybox-httpd
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Sergey Ponomarev <stokito@gmail.com>
+PKG_LICENSE:=ISC
+
+include $(INCLUDE_DIR)/package.mk
+
+# The BB httpd needs to be first enabled for compilation CONFIG_BUSYBOX_CONFIG_HTTPD=y.
```

Comment:
I decided to put the instruction here, once the package merged I'll copy it to wiki